scripts/in_container/run_ci_tests.sh

```diff
@@ -18,6 +18,10 @@
 # shellcheck source=scripts/in_container/_in_container_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_in_container_script_init.sh"
 
+echo "Starting the airflow triggerer"
```

Comment:
I modified this file because it `scripts/in_container/run_system_tests.sh` is executed only when `RUN_SYSTEM_TESTS` is `True`. It is unclear to me when it is the case. When the CLI `breeze testing tests ... --system` is run, `RUN_SYSTEM_TESTS` is `False`. See [source](https://github.com/apache/airflow/blob/main/Dockerfile.ci#L1094)